tests/app/vertx-opentracing/account-manager/src/main/resources/vertx-default-jul-logging.properties

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+handlers=java.util.logging.ConsoleHandler,java.util.logging.FileHandler
```

Comment:
For a production-quality application, this could indeed be a good idea. But for this example, I'd rather leave it there. I tried to quickly put things in `commons`, and I got the default logging from JUL.